tests/test_commands.py

```diff
@@ -680,9 +680,14 @@ def start_requests(self):
         )
         return []
 """
+        with open(os.path.join(self.cwd, "example.json"), "w") as file:
```

Comment:
`file` is a built-in in Python, should we change it to something like `fp`?

Reply:
Ok. I renamed it.